ui-core/src/components/inputs/Input.tsx

```diff
@@ -43,6 +44,8 @@ export type InputProps = React.InputHTMLAttributes<HTMLInputElement> &
     leadingContent?: InputAffixContent | InputAffixContentWithCallback;
     trailingContent?: InputAffixContent | InputAffixContentWithCallback;
     inputFieldWrapperClassname?: string;
+    showChevron?: boolean;
```

Comment:
I think the prop should be named withChevronButton. And this prop should be automatically set to true when we pass a non empty suggestion array to input with suggestion.